### PR TITLE
Add schwifty migration

### PR DESCRIPTION
**Pull request by wswoodruff**

### CODE REVIEW

**Comment by wswoodruff** [comment]
Needs tests to be written

**Comment by wswoodruff** [comment]
Reopening to show the diff

File: lib/index.js [hunk 1]
```diff
@@ -155,15 +163,50 @@ internals.initialize = (server, next) => {
...
+                return allModels[modelName];
+            });
+
+            return newKnexGroup;
+        });
+
+        // Maybe run schwifty-migration to generate migration file
+
+        if (collector.makeMigrationMode === true) {
+            SchwiftyMigration.genMigrationFile({ migrationGroups }, () => {
```
Comment (devinivy) [inline lib/index.js]: It's more hapi-stylistic (https://eslint.org/docs/rules/no-else-return), but I'd suggest instead returning here and nixing the `else {}` clause.

File: lib/index.js [hunk 2]
```diff
@@ -155,15 +163,50 @@ internals.initialize = (server, next) => {
...
+
+                process.exit(0);
```
Comment (devinivy) [inline lib/index.js]: schwifty-migration should also indicate an error and let its consumer (schwifty, in this case) `process.exit(1)`, or whatever else may be the appropriate reaction.

File: lib/index.js (hunk 1 above)
Comment (devinivy) [inline lib/index.js]: It never errors? :trollface:

File: lib/index.js (hunk 2 above)
Comment (devinivy) [inline lib/index.js]: We should investigate if it's kosher to call `server.stop()` during init.  If so, that would be a nice thing to do.

**Comment by wswoodruff** [comment]
Going a different route with schwifty-migration in the hapi-pal-verse!

**Pull request closed without merging**